client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx

```diff
@@ -50,11 +115,13 @@ class AutoRenewDisablingDialog extends Component {
 				<h2 className="auto-renew-disabling-dialog__header">{ translate( 'Before you go…' ) }</h2>
 				<p>{ description }</p>
 				<Button onClick={ onClose } primary>
-					{ translate( 'OK' ) }
+					{ translate( 'Confirm cancellation' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Confirm Cancellation' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).